packages/preferences-controller/src/PreferencesController.test.ts

```diff
@@ -36,6 +36,7 @@ describe('PreferencesController', () => {
         acc[curr] = true;
         return acc;
       }, {} as { [chainId in EtherscanSupportedHexChainId]: boolean }),
+      smartTransactionsOptInStatus: false,
```

Comment:
Why `smartTransactionsOptInStatus` as opposed to `smartTransactionsEnabled` (latter being more consistent with existing feature-toggling conf fields)

Reply:
Chose to use Opt In to represent users making a choice to opt in. If this toggle is true, Smart Transactions (STX) will be used _when_ possible.

When might Smart Transactions not be used?

If health checks on the Transaction API fail or some other checks fail, the user could still be _opted into_ STX, but STX will not be used due to those checks failing.

Reply:
I feel like `opt-in` and `enabled` give roughly the same information about whether STX may or may not still be used. 

I agree with @legobeat that `smartTransactionsEnabled` would be more consistent, especially because `smartTransactionOptInStatus` uses a Hungarian notation (where `Status` tries to describe the type of variable), but I don't have a strong opinion here

Reply:
Though, the extension's PreferencesController already uses `smartTransactionsOptInStatus`, so from this point of view I would say it is consistent across clients

Reply:
Sounds like we're good with `smartTransactionsOptInStatus` then, resolving.